main/app.py

```diff
@@ -0,0 +1,372 @@
+from __future__ import print_function
+import random
+import sqlite3
+import sys
+import tkFileDialog
+from Tkinter import Tk
+
+from clint.textui import colored, puts
+from colorama import init, Back, Style  # Fore
+from pyfiglet import figlet_format
+from termcolor import cprint
+
+from models.room import Office, Living_space
+from util.File import FileParser
+import ipdb
+
+
+
+class Amity(object):
+    """Description for amity"""
+
+    def __init__(self):
+        # variable used in create_rooms
+        self.rooms = {
+            'O': [],
+            'L': []
+        }
+        # variable used in add_person
+        self.people_data = {'Staff': [],
+                            'Fellow': []
+                            }
+        self.allocate = {
+            'O': {' ': []},
+            'L': {' ': []}
+        }
+        # variable used in allocation
+        self.offices = []
+        self.full_name = []
+        self.allocated_room = []
+
+        # variables used in add_person
+        self.name = ""
+        name = self.name
```

Comment:
**Code Qulaity: Nitpick**
You could make use of multiple assignments here to shorten code. Also on line 44/45.

```py
name = self.name = ""
```